packages/react-components/react-accordion/src/stories/Accordion/AccordionWithIcon.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { RocketRegular } from '@fluentui/react-icons';
 
-import { AccordionItem, AccordionHeader, AccordionPanel, Accordion } from '@fluentui/react-accordion';
+import { AccordionItem, AccordionHeader, AccordionPanel, Accordion } from '@fluentui/react-components';
```

Comment:
_nit:_ Order these alphabetically

```suggestion
import { Accordion, AccordionHeader, AccordionItem, AccordionPanel } from '@fluentui/react-components';
```